pyigm/guis/igmguesses.py

```diff
@@ -742,6 +767,13 @@ def update_model(self):
             return
         # Setup lines
         wvmin, wvmax = np.min(self.spec.wavelength), np.max(self.spec.wavelength)
+        spwvmin = self.parent.wvmin
+        spwvmax = self.parent.wvmax
+        if spwvmin is not None:
```

Comment:
I don't think `spwvmin` can ever be `None`.
Isn't it set by default to `min(spec.wavelength)` ?

If so, is this doing what you want it to do?

Reply:
I agree that spwvmin could not be None. Yes, the code was doing what I wanted it to do, but there was no need to write if spwvmin is not None, because it will always be true. Correcting (deleted if spwvmin is not None, and similar).